src/ui/layout/row-layout.svelte

```diff
@@ -6,49 +6,17 @@
   export let children: Snippet;
 </script>
 
-<div class="rcb-column gap-{gap} align-{align}">
```

Comment:
Since the existing row-layout file was actually a column layout, I moved that to a new col-layout file and create a new row-layout file. Opted not to include the alignment and gaps as they aren't currently used anywhere.